reemgawad/brunchtime
# Creating past branches and favourites pages

What changed:
-Visits controller: new method
-New route
-New view

How to test:
-GOTO browser home/brunch history-->should list all past visits
-Then go on a restaurant-->create a new visit-->go to brunch history (the latest should be at top of page)
